project/Settings.scala

```diff
@@ -214,28 +246,71 @@ object Settings extends Build {
     publish in (IntegrationTest,packageBin) := ()
   )
 
+  def makeTestGroups(tests: Seq[TestDefinition]) = {
```

Comment:
for a complex method like this an explicit return type would improve readability